blocks/article-hero-video/article-hero-video.js

```diff
@@ -0,0 +1,71 @@
+import { readBlockConfig } from '../../scripts/lib-franklin.js';
+
+export default function decorate(block) {
+  const data = readBlockConfig(block);
```

Comment:
When you look at https://31-magazine-article-hero-block-with-video--24life--hlxsites.hlx.page/drafts/ashrigondeka/the-change-master-john-berardis-second-act-will-help-you-too
the title  is currently `<title>Article Hero video</title>`. This is because there is no H1 in the Word doc, and Metadata does not set title. 
I suggest to format the title as Heading 1 in the Word document.

Reply:
done